### PR TITLE
Development

### PR DESCRIPTION
**Pull request by IvanDerlich**
- Setup: npm,webpack, eslint
- API query
- DOM Manipulation
- Styling

### CODE REVIEW

**Comment by IvanDerlich** [comment]
IDK why I have these stickler errors.
eslint runs through all the files with no errors.

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    feelsDiv.innerHTML = "";
+    cityLabel.innerHTML = cityInput.value;
+    tempDiv.innerHTML = "Fetching data..."
+    const queryUrl = `${url}&q=${city}`;
+
+    fetch(queryUrl).
+        then((response) => response.json()).
+        then((response) => {
+
+            console.log(response)
```
Comment (maelfosso) [inline src/index.js:31]: Remove it, please

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            humidityDiv.innerHTML = `Humidity: ${response.main.humidity} %`
+            feelsDiv.innerHTML = `Feels: ${response.main.feels_like} ºC`
+            cityLabel.innerHTML = cityInput.value;
+            errorDiv.innerHTML = "";
+
+        }).
+        catch((err) => {
+
+            errorDiv.innerHTML = err;
+            console.log(err)
```
Comment (maelfosso) [inline src/index.js:44]: Remove it, please

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
+/* eslint-disable prefer-named-capture-group */
+/* eslint-disable sort-keys */
+/* eslint-disable require-unicode-regexp */
+/* eslint-disable no-undef */
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const CleanWebpackPlugin = require("clean-webpack-plugin");
+
+
+config = {
```
Comment (maelfosso) [inline webpack.config.js:10]: You've forgotten `var`

File: .eslintrc.json [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+        "unicode-bom": [
+            "error",
+            "never"
+        ],
+        "valid-jsdoc": "error",
+        "vars-on-top": "error",
+        "wrap-iife": "error",
+        "wrap-regex": "error",
+        "yield-star-spacing": "error",
+        "yoda": "error"
```
Comment (maelfosso) [inline .eslintrc.json:248]: That file is not the one from [`Microverse`](https://github.com/microverseinc/linters-config/tree/master/javascript).
Update it, please.

File: webpack.config.js (hunk 3 above)
Comment (IvanDerlich) [reply webpack.config.js:10]: It gives me an eslint error

**Pull request merged**